doc/classes/MultiplayerAPI.xml

```diff
@@ -61,10 +61,21 @@
 			</return>
 			<description>
 				Method used for polling the MultiplayerAPI.
-				You only need to worry about this if you are using [memeber Node.custom_multplayer] override.
+				You only need to worry about this if you are using [member Node.custom_multplayer] override.
```

Comment:
Wouldn't it be *custom_multiplayer_api*?
Also, why aren't all those descriptions not in *latest* generated (readthe)docs?

Reply:
We had a discussion about that in #17227 and decided to use `multiplayer` instead of `multiplayer_api` to make the name shorter. That change though were only merged in #18709.

I think readthedocs is synced periodically in [another branch](https://github.com/godotengine/godot-docs), that change is too now :)

Reply:
then it should be `custom_multiplayer` and not `custom_multplayer`